lib/possible_bipartition.rb

```diff
@@ -1,4 +1,33 @@
+dislikes = [[],
+            [2, 3],
+            [1, 4],
+            [1],
+            [2]]
 
 def possible_bipartition(dislikes)
```

Comment:
This solution will work, as long as the graph is totally connected.  If the graph has 2 or more unconnected sub-graphs, then it won't work.  That's actually why I had node 0 unconnected to the rest of the graph in the tests.